apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -28,23 +29,22 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 	);
 
 	return (
-		<section className={styles["signup-container"]}>
-			<div className={styles["logo-container"]}>
-				<div
-					className={getValidClassNames(
-						styles["circle"],
-						styles["circle-pink"],
-					)}
-				/>
-				<h1>LOGO</h1>
-			</div>
-			<h3 className={styles["title-form"]}>CREATE AN ACCOUNT</h3>
-			<span className={styles["auth-info"]}>
-				Already have an account? Go to
-				<Link to={AppRoute.SIGN_IN}> Sign in</Link>
-			</span>
-			<form onSubmit={handleFormSubmit}>
-				<div className={styles["input-groups"]}>
+		<div className={styles["container"]}>
+			<div className={styles["form-container"]}>
+				<form className={styles["form"]} onSubmit={handleFormSubmit}>
+					<div className={styles["form-header"]}>
+						<div className={styles["form-header__logo-container"]}>
+							<div className={styles["form-header__logo"]} />
+							<span className={styles["form-header__logo-text"]}>LOGO</span>
+						</div>
+
+						<h1 className={styles["form-header__text"]}>CREATE AN ACCOUNT</h1>
```

Comment:
On my PR, I was instructed to use 	`text-transform: uppercase;` in CSS for cases like this one. Using this rule, you can provide a lowercase text in JSX and CSS will transform it to upper case.

Reply:
but, i see in the repo now used upper case text directly.

Reply:
We should update to `text-transform: uppercase`